tmux/formater_test.go

```diff
@@ -2,11 +2,83 @@ package tmux
 
 import (
 	"io"
+	"strings"
 	"testing"
 
 	"github.com/arl/gitstatus"
 )
 
+// expandPlaceholders converts readable placeholder strings to actual test strings
+func expandPlaceholders(s string) string {
+	// Style placeholders
+	s = strings.ReplaceAll(s, "[style:clear]", "[style:clear]")
+	s = strings.ReplaceAll(s, "[style:state]", "[style:state]")
+	s = strings.ReplaceAll(s, "[style:branch]", "[style:branch]")
+	s = strings.ReplaceAll(s, "[style:remote]", "[style:remote]")
+	s = strings.ReplaceAll(s, "[style:divergence]", "[style:divergence]")
+	s = strings.ReplaceAll(s, "[style:staged]", "[style:staged]")
+	s = strings.ReplaceAll(s, "[style:conflict]", "[style:conflict]")
+	s = strings.ReplaceAll(s, "[style:modified]", "[style:modified]")
+	s = strings.ReplaceAll(s, "[style:untracked]", "[style:untracked]")
+	s = strings.ReplaceAll(s, "[style:stash]", "[style:stash]")
+	s = strings.ReplaceAll(s, "[style:clean]", "[style:clean]")
+	s = strings.ReplaceAll(s, "[style:insertions]", "[style:insertions]")
+	s = strings.ReplaceAll(s, "[style:deletions]", "[style:deletions]")
+	
+	// Symbol placeholders
+	s = strings.ReplaceAll(s, "[symbol:branch]", "[symbol:branch]")
+	s = strings.ReplaceAll(s, "[symbol:hash]", "[symbol:hash]")
+	s = strings.ReplaceAll(s, "[symbol:ahead]", "[symbol:ahead]")
+	s = strings.ReplaceAll(s, "[symbol:behind]", "[symbol:behind]")
+	s = strings.ReplaceAll(s, "[symbol:staged]", "[symbol:staged]")
+	s = strings.ReplaceAll(s, "[symbol:conflict]", "[symbol:conflict]")
+	s = strings.ReplaceAll(s, "[symbol:modified]", "[symbol:modified]")
+	s = strings.ReplaceAll(s, "[symbol:untracked]", "[symbol:untracked]")
+	s = strings.ReplaceAll(s, "[symbol:stash]", "[symbol:stash]")
+	s = strings.ReplaceAll(s, "[symbol:clean]", "[symbol:clean]")
+	s = strings.ReplaceAll(s, "[symbol:insertions]", "[symbol:insertions]")
+	s = strings.ReplaceAll(s, "[symbol:deletions]", "[symbol:deletions]")
+	
+	return s
+}
```

Comment:
this list of replaces is totally bogus...
you replace a string by itself, each time
Remove this function, and all other helper functions, they're not needed for the task at hand